### PR TITLE
Lovelace: correct vertical stack

### PR DESCRIPTION
**Pull request by c727**
**Description:**

correct config, improve text

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_lovelace/vertical-stack.markdown [hunk 1]
```diff
@@ -10,9 +10,7 @@ sharing: true
 footer: true
 ---
 
-Stack card will allow you to stack together multiple cards so they always sit together in the same column one on top of the other. Keep in mind this can be used with any cards and even used alongside a [horizontal-stack](/lovelace/horizontal-stack/).
-
-## {% linkable_title Options %}
+Vertical stack allows you to group multiple cards to use the same column.
```
Comment (DubhAd) [inline source/_lovelace/vertical-stack.markdown]: Reads a little stilted, Maybe *Vertical stack allows you to group multiple cards so they always sit in the same column*?

File: source/_lovelace/vertical-stack.markdown [hunk 2]
```diff
@@ -21,24 +19,11 @@ type:
...
-      description: The type of the card to stack.
-      type: string
-    entity:
-      required: true
-      description: "An `entity_id` to use in the frontend."
-      type: string
-    image:
-      required: true
-      description: The URL to an image.
-      type: string
```
Comment (fabaff) [inline source/_lovelace/vertical-stack.markdown:38]: Those keys are still present in the example below. Not documenting them will at some point lead to confusion as they seems to be required.

**Review by fabaff — CHANGES_REQUESTED** [review]

File: source/_lovelace/vertical-stack.markdown (hunk 2 above)
Comment (c727) [reply source/_lovelace/vertical-stack.markdown:38]: These keys don't belong to this card. Instead they are part of the sub-cards

Comment (fabaff) [reply source/_lovelace/vertical-stack.markdown:38]: I guess then that I was the only one that go confused.

**Review by fabaff — APPROVED** [review]

**Pull request merged**